anaconda.py

```diff
@@ -490,8 +490,8 @@ def _earlyExceptionHandler(ty, value, traceback):
     if not conf.target.is_directory:
         from pyanaconda.ui.lib.storage import reset_storage
 
-        threadMgr.add(AnacondaThread(name=constants.THREAD_STORAGE,
-                                     target=reset_storage))
+        thread_manager.add(AnacondaThread(name=constants.THREAD_STORAGE,
+                                          target=reset_storage))
```

Comment:
The indentation of these calls drives me crazy, but that's up to you if you want to fix it.

Reply:
Ah, yes. Actually I did that on the oscap code when I changed it later, just didn't realize this was here too.